antevus-app/src/lib/prisma.ts

```
@@ -0,0 +1,21 @@
+import { PrismaClient } from '@prisma/client'
+import { logger } from '@/lib/logger'
+
+declare global {
+  var prisma: PrismaClient | undefined
+}
+
+export const prisma = global.prisma || new PrismaClient({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)